### PR TITLE
Added 2 new endpoints and some docstrings

### PR DESCRIPTION
**Pull request by bimusiek**
Hey, all tested and we use it already :) 
Docstrings are awesome in Pycharm because then you have nice autocompletion, would be good to rewrite all docs to it.
Also, I am not returning page count in new endpoints (that is coming in headers) because of construction of raw_request, so it requires further work if someone needs it.

### CODE REVIEW

**Comment by rschlaikjer** [comment]
Hi bimusiek, thanks for the functionality! Would you mind updating the tests? I think they will fail with the new params option in `_raw_request`.
It also looks like you have a couple of lint errors - you can check both of these by running 

```
tox -e py27-test,lint
```

File: LayerClient/LayerClient.py [hunk 1]
```diff
@@ -534,6 +610,10 @@ def from_dict(dict_data):
             dict_data.get('is_unread'),
         )
 
+    @classmethod
+    def from_list(cls, list_data):
```
Comment (rschlaikjer) [inline LayerClient/LayerClient.py:638]: For consistency, can you use `self` here and not `cls`?

File: LayerClient/LayerClient.py [hunk 2]
```diff
@@ -811,6 +891,10 @@ def from_dict(dict_data):
             dict_data.get('metadata'),
         )
 
+    @classmethod
+    def from_list(cls, list_data):
```
Comment (rschlaikjer) [inline LayerClient/LayerClient.py]: Same here

File: LayerClient/LayerClient.py (hunk 1 above)
Comment (rschlaikjer) [reply LayerClient/LayerClient.py:638]: Ah, silly me, missed that this was a class method. Never mind

File: LayerClient/LayerClient.py [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
```
Comment (rschlaikjer) [inline LayerClient/LayerClient.py:2]: Are you running this on python < 2.7? I think absolute import is the default now

Comment (bimusiek) [reply LayerClient/LayerClient.py:2]: We are working on 2.7 but as I understood only version 3 has absolute imports mandatory. (http://stackoverflow.com/a/11741846/2249584)

**Comment by bimusiek** [comment]
Tests updated but I am not asserting params anywhere. I will see if I have time to write some tests for this 2 endpoints.

**Comment by bimusiek** [comment]
I also added endpoint to send receipts to messages. The one that was there before never worked (as Layer does not support this endpoint since long time).

File: LayerClient/LayerClient.py [hunk 4]
```diff
@@ -124,13 +132,13 @@ def _get_layer_uri(self, *suffixes):
 
         Return: A complete URI for an endpoint with optional arguments
         """
-        suffix_string = '/'.join(suffixes) if suffixes else ''
         return 'https://api.layer.com/apps/{app_id}/{suffix}'.format(
             app_id=self.app_uuid,
-            suffix=suffix_string,
+            suffix='/'.join(map(unicode, suffixes)),
```
Comment (rschlaikjer) [inline LayerClient/LayerClient.py]: Just noticed that this breaks python 3 compatibility - 

```
tox -e py3-test
[...]
>           suffix='/'.join(map(unicode, suffixes)),
        )
E       NameError: name 'unicode' is not defined
```

Are there any cases where non-unicode suffixes are breaking API calls? If there are, I guess we need a helper method that can call `str()` for python 3 and `unicode()` for python 2

**Comment by rschlaikjer** [comment]
Nice, thanks for fixing read receipts. We've stopped using layer internally, so we've been a bit slow on the uptake for API changes.

**Comment by bimusiek** [comment]
I think it is fixed. We will be adding more and more features to Layer over the months as it is almost our core service :) 

If I can ask, what do you use for chat integration now? Layer seems like perfect fit to have customizable chat for us, but if there is something better I would take a look :)

**Comment by rschlaikjer** [comment]
Yup, tests look good! I'll bump the version and get this up on pypi. 

We've actually just stopped using chat altogether in our product, so no recommendations I'm afraid.

**Comment by rschlaikjer** [comment]
Published to pypi as version 0.1.10. Thanks again for the patch!
https://pypi.python.org/pypi/LayerClient/0.1.10

**Pull request merged**